dnd/.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # dependencies
 /node_modules
```

Comment:
We are still commiting `node_modules` to repo. Try like so `node_modules` without the `/`